src/MockPublisher.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of the Mercure Component project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Symfony\Component\Mercure;
+
+final class MockPublisher implements PublisherInterface
+{
+    private $callable;
+
+    /**
+     * @param (callable(Update): string) $callable
+     */
+    public function __construct(callable $callable)
+    {
+        $this->callable = $callable;
+    }
+
+    public function publish(Update $update): string
+    {
+        return ($this->callable)($update);
+    }
+
+    /**
+     * @deprecated since 0.5, use {@see PublisherInterface::publish} instead.
+     */
+    public function __invoke(Update $update): string
```

Comment:
Is it useful to introduce a new method which is already deprecated?

Reply:
This method still exist in the interface, so we have to implement it.